cves/kernel/CVE-2016-5728.yml

```diff
@@ -335,9 +355,12 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: False
+  stacktrace_with_fix: False
+  note: | 
+    No stacktraces were found, however there might have been one since the original 
+    person that found the fix knew specific line numbers that pointed out the issues in the 
```

Comment:
Good observation here.  They may have also debugged the kernel themselves before reporting.